tesseract_motion_planners/ompl/include/tesseract_motion_planners/ompl/profile/ompl_profile.h

```diff
@@ -31,6 +31,7 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <vector>
 #include <memory>
 #include <Eigen/Geometry>
+#include <yaml-cpp/yaml.h>
```

Comment:
In all of the profiles can you move this to the cpp and forward declare the node as shown below.
``` c++
namespace YAML
{
class Node;
}
```